gilbreth_perception/CMakeLists.txt

```diff
@@ -43,11 +49,10 @@ find_package(catkin REQUIRED)
 ##   * add every package in MSG_DEP_SET to generate_messages(DEPENDENCIES ...)
 
 ## Generate messages in the 'msg' folder
-# add_message_files(
-#   FILES
-#   Message1.msg
-#   Message2.msg
-# )
+ add_message_files(
+   FILES
+   ObjectDetection.msg
```

Comment:
Please use the messages in **gilbreth_msgs**, you can change those message there as well.